chesster.js

```diff
@@ -1,13 +1,22 @@
+// extlibs
 var async = require("async");
 var Botkit = require('botkit');
```

Comment:
There shouldn't be any impact as far as I know.  

If we get to a point where it's possible to remove the botkit reference in chesster.js, then by all means let's do it. However, that won't be my goal within this pull request.  As i don't want this branch to live for too long without being merged.